db/schema.rb

```diff
@@ -970,6 +1006,16 @@
     t.index ["status_id"], name: "index_status_stats_on_status_id", unique: true
   end
 
+  create_table "status_trend_highlights", force: :cascade do |t|
```

Comment:
Leftover from branch/rebase elsewhere? (also see terms_of_services and user_agreements below for same)